src/main/java/com/wellsky/workflow/WorkflowApplication.java

```diff
@@ -7,6 +7,7 @@
 public class WorkflowApplication {
 
     public static void main(String[] args) {
+        String SuperHorrible = "";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://hhh-aks.eastus.cloudapp.azure.com/coding_rules#rule_key=squid%3AS00117)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "SuperHorrible" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://hhh-aks.eastus.cloudapp.azure.com/coding_rules#rule_key=squid%3AS1481)